test/python/circuit/test_controlled_gate.py

```diff
@@ -923,11 +925,12 @@ def test_open_controlled_unitary_matrix(self, num_ctrl_qubits):
     def test_open_control_cx_unrolling(self):
         """test unrolling of open control gates when gate is in basis"""
         qc = QuantumCircuit(2)
-        qc.cx(0, 1, ctrl_state=0)
+        qc.append(U3Gate(np.pi, 0, np.pi), [0])
+        qc.cx(0, 1)
+        qc.append(U3Gate(np.pi, 0, np.pi), [0])
```

Comment:
This test doesn't seem to test what it was supposed to anymore: it should test that an open control CX gets decomposed into a closed control CX and X (or U) gates. What is the output it you apply the basis translator to the open control CX? The same question holds for the other open control tests below 🙂

Reply:
I agree, the test does not seem to test what it was suppose to do anymore and I was a bit confused how to switch from Unroller to BasisTranslator in these cases. 

The output when applying the bassis translator to the open control CX is:

`qiskit.transpiler.exceptions.TranspilerError: "Unable to translate the operations in the circuit: ['cx_o0'] to the backend's (or manually specified) target basis: ['measure', 'delay', 'barrier', 'snapshot', 'cx', 'reset', 'u3']. This likely means the target basis is not universal or there are additional equivalence rules needed in the EquivalenceLibrary being used. For more details on this error see: https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.transpiler.passes.BasisTranslator#translation-errors"`

Reply:
Hmm yes the basis translator alone is not able to handle this right now, but then we can also not state that it is able to replace the unroller. Off the top of my head, we could either
* extend the basis translator to handle open controls (it should detect the `_o*` ending and apply corresponding `X` gates)
* use `transpile(circuit, basis_gates=..., optimization_level=0)` instead of the basis translator
* use the `generate_translation_passmanager` function to get a pass manager for translation

For this PR, the `transpile` is certainly the easiest approach, but let's double check with the rest of the team.